docs/source/CUSTOM-JOB.rst

```diff
@@ -286,30 +286,68 @@ Generally, the consumer for each Redis queue is scaled relative to the amount of
 
     <tt><a href="https://github.com/vanvalenlab/kiosk/blob/master/conf/patches/redis-exporter-script.yaml">/conf/patches/redis-exporter-script.yaml</a></tt>
 
-Connecting custom consumers with the frontend
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Connecting custom consumers with the Kiosk
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Finally, in order to use the frontend interface to interact with your new consumer, you will need to add the new queue to the |kiosk-frontend|.
+A number of Kiosk components will need the new queue name in order to fully integrate the new job.
 
-In the |kiosk-frontend| helmfile (|frontend.yaml|), add or modify the ``env`` variable ``JOB_TYPES`` and replace with :data:`consumer_type`.
+1. |frontend.yaml|
 
-.. code-block:: yaml
+   In the |kiosk-frontend| helmfile (|frontend.yaml|), add or modify the ``env`` variable ``JOB_TYPES`` and replace with :data:`consumer_type`.
 
-    env:
-        JOB_TYPES: "segmentation,tracking,<new job name>"
+   .. code-block:: yaml
+
+       env:
+           JOB_TYPES: "segmentation,tracking,<new job name>"
+
+2. |redis-janitor.yaml|
+
+   The |kiosk-redis-janitor| monitors queues in an ``env`` variable ``QUEUES`` for stalled jobs, and restarts them. :data:`consumer_type` must be added here as well.
+
+   .. code-block:: yaml
+
+       env:
+           QUEUES: "segmentation,tracking,<new job name>"
+
+3. |autoscaler.yaml|
+
+  The |kiosk-autoscaler| also has an ``env`` variable ``QUEUES`` which it uses to determine whether a GPU must be activated. Add :data:`consumer_type` to this variable too.
+
+  .. code-block:: yaml
```

Comment:
Add one additional space to the indentation of these lines. The leftmost character should line up the first character after the number.